js/app/controllers/views/settingsviewcontroller.js

```diff
@@ -322,6 +322,33 @@ angular.module('Music').controller('SettingsViewController', [
 			});
 		};
 
+		$scope.generateSession = function() {
```

Comment:
This function and $scope.clearSession should be renamed to contain the term "scrobble" or "scrobbling" to make it obvious to any reader, what they are about